portal/code/cx-portal/src/components/Router.tsx

```diff
@@ -37,6 +44,17 @@ const plainRoutes: { [page: string]: JSX.Element } = {
 }
 
 export default function Router() {
+  const dispatch = useDispatch()
```

Comment:
I would remove this change because the problem in index.tsx can be solved easier or has already been solved. Look an my comment there.